### DISCUSSION TITLE
I've been verified for a few days and have not received my benefits any help I would be thankful for to speed up the process. If not I'll keep waiting

### DISCUSSION THREAD

**Discussion by AlistairMaul** [community/community#144685] [GitHub Education]
### Hi everyone,

I've been verified for a few days and have not received my benefits any help I would be thankful for to speed up the process. If not I'll keep waiting

**Comment by davevad93** [comment]
Hi @AlistairMaul ,

GitHub no longer provides manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AlistairMaul was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
